lib/src/local_working_copy.rs

```diff
@@ -808,12 +808,12 @@ impl TreeState {
             matcher: fsmonitor_matcher,
             watchman_clock,
         } = self.make_fsmonitor_matcher(fsmonitor_settings)?;
-        let fsmonitor_matcher = match fsmonitor_matcher.as_ref() {
+        let chosen_matcher = match fsmonitor_matcher.as_ref() {
```

Comment:
I renamed this because the previous name was probably just a copy/paste from a refactoring, and rather confusing.